clients/cloud/python/README.md

```diff
@@ -0,0 +1,55 @@
+# Overview
+
+Produce messages to and consume messages from Confluent Cloud using Confluent Python Client for Apache Kafka
+
+
+# Prerequisites
+
+* Install Confluent's Python Client for Apache Kafka: https://github.com/confluentinc/confluent-kafka-python
+* File with client configuration parameters to connect to your Confluent Cloud instance
```

Comment:
Link to instructions on how to get those? Especially the thing where there are tabs for different languages and that you need the C (librdkafka) one even though this is a python example.

Reply:
Ack